src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -80,6 +85,26 @@
         border: none;
         background-color: inherit;
         font-size: inherit;
+
+        .dropdown-item {
+          &:hover,
+          &:focus {
+            // In the mobile sidebar, the dropdown menu is inlined with the
+            // other links, which do not have background-color changes on hover
+            // and focus
+            background-color: unset;
+          }
+        }
```

Comment:
Something I observed on the PR build of our site: when the viewport is narrow (i.e., mobile device), the extra items are no longer in a "more" dropdown (because the nav items are now arranged vertically, not horizontally).  That much is expected and correct. Also expected and correct is that `Tab` and `Shift-Tab` navigation through those all nav items works seamlessly. What was not expected was that any items that *would have been* in the "more" dropdown in the wide viewport are navigable with up-arrow and down-arrow *even in in the narrow viewport* despite the fact that they're no longer displayed in a drop-down menu. The "non-more-dropdown" nav items are not navigable with up/down arrows.

I don't think this is quite a "bug" exactly --- all nav items are navigable by Tab and Shift-Tab as they should be. But it's odd that only *some* nav items have this extra means of navigating and *when in a narrow viewport, there is no way to tell them apart from the other nav items that don't allow up/down navigating*.

Feel free to "wontfix" this, or migrate it to a separate issue to be addressed later.

Reply:
I believe this is related to https://github.com/pydata/pydata-sphinx-theme/issues/1603.

It might be worth following up some of the discussion in #1604, so that we can have a fix that is less of a hack.